### PR TITLE
Changed deprecated keys and used cryptosigner

### PR DESCRIPTION
**Pull request by DarikshaAnsari**
"Fixed issues #73 , #74 , #68  "

changed in readme.md and run_demo.py
in-toto-run with --key is now --signing-key
in-toto-record with --key is now --signing-key
in-toto-verify with --layout-keys is now  --verification-keys

Used CryptoSigner instead of signer in create_layout.py and added module requirement in requirements.txt

### CODE REVIEW

**Comment by DarikshaAnsari** [comment]
@lukpueh I have changed all the files you've mentioned , PTAL !!

**Review by lukpueh — CHANGES_REQUESTED** [review]
Thanks!! This is almost complete. Can you please check the [ci log](https://github.com/in-toto/demo/actions/runs/9123811857/job/25088867862?pr=75) and fix the errors? You can also run the test locally:
```sh
 python3 run_demo_md.py
```

I few hints:
- The functionary keys in `create_layout.py` (key_bob and key_carl) are no longer key dictionaries but `Key` objects now. You need to slightly adopt the way they are used in the rest of the script.
- If you do change the commands in `README.md`, you'll also need to update the expected output in `run_demo_md.py`. But you can also leave that for a separate PR, as we discussed.

File: owner_alice/create_layout.py [hunk 1]
```diff
@@ -1,16 +1,39 @@
...
+    crypto_public_key = load_pem_public_key(data)
+    key = SSlibKey.from_crypto(crypto_public_key)
+
+    # Create a key_dict, which is accepted by `verifylib.in_toto_verify` or `Metablock.verify_signature`
+    # NOTE: securesystemslib and in-toto key dicts differ:
+    # the former don't include the keyid, which the latter require
+    key_dict = key.to_dict()
+    key_dict["keyid"] = key.keyid
+
+    return key_dict
```
Comment (lukpueh) [inline owner_alice/create_layout.py:23]: IMO it's okay to just import this internal method from in_toto:

```
# https://github.com/in-toto/in-toto/issues/663
from in_toto.models._signer import _load_public_key_from_file
```

**Comment by lukpueh** [comment]
Hm. Tests are still failing. Feel free to debug, but this might be a bit tricky. I am happy to take over.

**Review by lukpueh — APPROVED** [review]
Nice troubleshooting with the keyid change!! Thanks

**Comment by DarikshaAnsari** [comment]
Thank you @lukpueh working with this issue is so interesting.

**Pull request merged**